modules/imgcodecs/include/opencv2/imgcodecs/macosx.h

```diff
@@ -0,0 +1,58 @@
+
+/*M///////////////////////////////////////////////////////////////////////////////////////
+//
+//  IMPORTANT: READ BEFORE DOWNLOADING, COPYING, INSTALLING OR USING.
+//
+//  By downloading, copying, installing or using the software you agree to this license.
+//  If you do not agree to this license, do not download, install,
+//  copy or use the software.
+//
+//
+//                          License Agreement
+//                For Open Source Computer Vision Library
```

Comment:
Please use short license header: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#file-structure

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

Reply:
🙆‍♂️

Reply:
Thank you for updates!
Please make similar update of license headers of new files in this PR.
Please remove empty first line in this file.

Reply:
https://github.com/opencv/opencv/pull/18547/commits/ae0ba082e222870f3a850c115c874e36575b321e